bolt-modules/prompt/lib/puppet/functions/prompt.rb

```diff
@@ -30,14 +34,20 @@ def prompt(prompt, options = {})
                               action: 'prompt')
     end
 
-    options = options.transform_keys(&:to_sym)
-
+    options  = options.transform_keys(&:to_sym)
     executor = Puppet.lookup(:bolt_executor)
+
     # Send analytics report
     executor.report_function_call(self.class.name)
 
+    # Require default to be a string value
+    if options.key?(:default) && !options[:default].is_a?(String)
+      raise Bolt::ValidationError, "Option 'default' must be a string"
+    end
+
     response = executor.prompt(prompt, options)
 
+    # If sensitive, wrap it
```

Comment:
An example of this would also be great, since you can't set the default to an actual `Sensitive` type and I expect this will be a pretty common use case.